utils/file_utils.jsx

```diff
@@ -35,3 +35,26 @@ export function trimFilename(filename) {
 
     return trimmedFilename;
 }
+
+export function getFileTypeFromMime(extension) {
```

Comment:
Do we need to duplicate this logic? Could we just use the method that gets this from file extension?

Reply:
What do you mean by duplicating logic do you see this anywhere else in the code?. I don't think we have an existing way of decoding a file type from browser inferred mime type. All we have are types decoded from server formatted mime types which will be different from browser mime types.

Reply:
I mean can we use the existing method that just uses the file extension instead of adding a new one that uses the mime type?

Reply:
> I mean can we use the existing method that just uses the file extension instead of adding a new one that uses the mime type?

We will not have file extension on the browser. We have to use mimetype to figure out the possible extension of the file.

Hence this method is added as we don't have a way of figuring out extension on client.

> Actually, I think something is getting mixed up in this code.

Poor choice of variable name usage. I changed it to reflect mimetype every where. Let me know if that clears up the air for you.